### ISSUE TITLE
Y axis label not displayed in tooltip with multichart

### ISSUE THREAD

**Issue by AlexandreDenis** [krispo/angular-nvd3#415]
I am working with a multichart which has two plots. Each plot has it own y axis.
My chart options are the following :

```javascript
chart: {
   type: "multiChart",
   x: function (d) {
       return d.x;
   },
   y: function (d) {
       return d.y;
   },
   xAxis: {
       tickFormat: function (d) {
           return d3.time.format('%H:%M:%S')(new Date(d));
       }
    },
    yAxis1: {
        axisLabel: "Pressure",
        tickFormat: function (d) {
            return d3.format(".2")(d);
        }
    },
    yAxis2: {
         axisLabel: "Temperature",
         tickFormat: function (d) {
             return d3.format(".2")(d);
         }
    },
    tooltip: {
         headerFormatter: function (d) {
             return d3.time.format('%b %d %Y %H:%M:%S')(new Date(d));
         }
    },
    showLegend: true,
    showMinMax: true
}
```

Everything works fine execpt one thing about the tooltip. Indeed, the y axis label is not displayed as it sould be. Here is a screenshot about the issue :

![angular-nvd3 multichart tooltip](http://image.noelshack.com/fichiers/2016/16/1461231977-angular-nvd3-multichart-tooltip.png)

Is it something I missed ? Or is it a bug ?

**Comment by krispo** [comment]
Yep, it's a bug. 
Upgrade the version please, it seems to be fixed in nvd3 1.8.2.
I updated [demo](http://plnkr.co/edit/zsNxBJ?p=preview) plunk too.

**Comment by AlexandreDenis** [comment]
Works fine. Thank you again krispo.
